### PR TITLE
wemo: use non-deprecated channel UID constructor

### PR DESCRIPTION
**Pull request by maggu2810**
Signed-off-by: Markus Rathgeb <maggu2810@gmail.com>

### CODE REVIEW

File: extensions/binding/org.eclipse.smarthome.binding.wemo/src/main/java/org/eclipse/smarthome/binding/wemo/handler/WemoHandler.java [hunk 1]
```diff
@@ -207,9 +207,7 @@ public void onValueReceived(String variable, String value, String service) {
                         if (binaryState != null) {
                             logger.trace("New InsightParam binaryState '{}' for device '{}' received", binaryState,
                                     getThing().getUID());
-                            updateState(
-                                    new ChannelUID(getThing().getThingTypeUID(), getThing().getUID(), CHANNEL_STATE),
-                                    binaryState);
+                            updateState(new ChannelUID(getThing().getUID(), CHANNEL_STATE), binaryState);
```
Comment (kaikreuzer) [inline extensions/binding/org.eclipse.smarthome.binding.wemo/src/main/java/org/eclipse/smarthome/binding/wemo/handler/WemoHandler.java]: The cleanest code here would actually look like this:
```java
updateState(CHANNEL_STATE, binaryState);
```

Comment (maggu2810) [inline extensions/binding/org.eclipse.smarthome.binding.wemo/src/main/java/org/eclipse/smarthome/binding/wemo/handler/WemoHandler.java]: Will fix this later

**Comment by maggu2810** [comment]
Fixed

**Comment by kaikreuzer** [comment]
Thanks!

**Pull request merged**